pgjdbc/src/test/java/org/postgresql/test/jdbc2/StringTypeUnspecifiedArrayTest.java

```diff
@@ -20,6 +23,25 @@
 
 @RunWith(Parameterized.class)
 public class StringTypeUnspecifiedArrayTest extends BaseTest4 {
+
+  @Before
+  public void createExtension() {
+    try {
+      TestUtil.createExtension(con, "cube");
+    } catch (Exception ex) {
+      // we can ignore this as the test won't run if the extension isn't loaded
```

Comment:
This should probably be `assume(false)`, so the tests are skipped. Then, the test method does not need `assumeExtensionInstalled`